### DISCUSSION TITLE
The new Internationalized Routing is awesome, how can I use the domain routing?

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [vercel/next.js#18454] [Help] [answered]
Hello guys, great job at the internacionalization proccess... 

There is example about how to use domain internacionalization? And how can I integrate the locale auto detected to my i18n? And there is a way to load only the language "dictionary" that I will use?

Thanks so much guys!

**Comment by M1ck0** [comment, marked as answer]
You should read blog and docs

[Here](https://nextjs.org/blog/next-10#domain-routing) is a guide on domain routing.
[Here](https://nextjs.org/blog/next-10#language-detection) is a guide on language detection.
